tests/packagedcode/data/podspec/Starscream/Starscream.podspec.expected.json

```diff
@@ -0,0 +1,43 @@
+{
+  "type": "podspec",
+  "namespace": null,
+  "name": "Starscream",
+  "version": "4.0.3",
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Swift",
+  "description": null,
+  "release_date": null,
+  "parties": [
+    {
+      "type": null,
+      "role": "author",
+      "name": "Dalton Cherry, Austin Cherry",
+      "email": "http://daltoniam.com, http://austincherry.me",
+      "url": null
+    }
+  ],
+  "keywords": [],
+  "homepage_url": null,
+  "download_url": null,
+  "size": null,
+  "sha1": null,
+  "md5": null,
+  "sha256": null,
+  "sha512": null,
+  "bug_tracking_url": null,
+  "code_view_url": null,
+  "vcs_url": null,
+  "copyright": null,
+  "license_expression": "apache-2.0",
+  "declared_license": "Apache License",
+  "notice_text": null,
+  "root_path": null,
+  "dependencies": [],
+  "contains_source_code": null,
+  "source_packages": [],
+  "purl": "pkg:podspec/Starscream@4.0.3",
+  "repository_homepage_url": null,
+  "repository_download_url": null,
+  "api_data_url": null
+}
```

Comment:
Can we infer any of these three URLs?

Reply:
homepage_url is added but I am not sure about download_url and api_data_url